src/api/apiHelpers.native.test.ts

```diff
@@ -42,22 +43,42 @@ const respondWith = async (
   })
 }
 
-const accessToken = 'some fake access token'
+const accessToken =
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJmcmVzaCI6ZmFsc2UsImlhdCI6MTY4OTI1NjM1NywianRpIjoiNTFkMjc5OGMtZDc1YS00NjQ1LTg0ZTAtNTgxYmQ3NTQzZGY3IiwidHlwZSI6ImFjY2VzcyIsInN1YiI6ImJlbmVfMThAZXhhbXBsZS5jb20iLCJuYmYiOjE2ODkyNTYzNTcsImV4cCI6MTY4OTI1NzI1NywidXNlcl9jbGFpbXMiOnsidXNlcl9pZCI6MTI3MTN9fQ.OW09vfchjTx-0LfZiaAJu8eMd9aftExhxR4bUsgl3xw'
+const decodedAccessToken = {
+  fresh: false,
+  iat: 1689576398,
+  jti: 'a90f96de-185d-4edb-a878-d714eea7ff74',
+  type: 'access',
+  sub: 'bene_18@example.com',
+  nbf: 1689576398,
+  exp: 1689577298,
+  user_claims: {
+    user_id: 12713,
+  },
+}
+
 const apiUrl = '/native/v1/me'
 const optionsWithAccessToken = {
   headers: {
     Authorization: `Bearer ${accessToken}`,
   },
 }
 
+jest.spyOn(jwt, 'decodeAccessToken').mockReturnValue(decodedAccessToken)
+
 jest.spyOn(CodePush, 'getUpdateMetadata').mockResolvedValue(null)
 
+jest.useFakeTimers('legacy')
+
 describe('[api] helpers', () => {
   const mockFetch = jest.spyOn(global, 'fetch')
   const mockGetAccessTokenStatus = jest.spyOn(jwt, 'getAccessTokenStatus')
   const mockGetRefreshToken = jest.spyOn(Keychain, 'getRefreshToken')
   const mockClearRefreshToken = jest.spyOn(Keychain, 'clearRefreshToken')
 
+  afterEach(cleanRefreshedAccessToken)
```

Comment:
nitpick (non-blocking): on ne nettoie pas, on supprime / on enlève

```suggestion
  afterEach(removeRefreshedAccessToken)
```